0.16 - Getting a server error when attempting to preview a resource

## Observed behavior
I am getting a server error each time I attempt to preview a resource at Coach > Reports > Lesson > Report

## Expected behavior
It should be possible to preview the resource.

## Steps to reproduce the issue
1. Install the the following [0. 16 build ](https://buildkite.com/learningequality/kolibri-debian/builds/5491) 
2. Go to Coach > Reports > Lesson > Report
3. Click on the resource
4. Click the Preview button

## Video

https://user-images.githubusercontent.com/79847249/208930147-d7a410e0-f48a-46fb-86c3-03f4c4f4ddad.mp4

## Error logs:

```
Kolibri Modules: kolibri.plugins.coach.app registered
vue.runtime.esm.js:1897 Error: Input data should be a String
    at MarkdownIt.parse (index.js:519:11)
    at MarkdownIt.render (index.js:544:36)
    at Sub.description (index.vue:222:1)
    at Watcher.get (vue.runtime.esm.js:4495:25)
    at Watcher.evaluate (vue.runtime.esm.js:4597:21)
    at Sub.description (vue.runtime.esm.js:4851:17)
    at fn (index.vue?95c6:114:21)
    at Object.normalized [as header] (vue.runtime.esm.js:2605:62)
    at Sub.MultiPaneLayoutvue_type_template_id_43808cfe_scoped_true_render (MultiPaneLayout.vue?23e6:8:18)
    at vue-composition-api.mjs:1942:35
logError @ vue.runtime.esm.js:1897
globalHandleError @ vue.runtime.esm.js:1888
handleError @ vue.runtime.esm.js:1848
Vue._render @ vue.runtime.esm.js:3571
updateComponent @ vue.runtime.esm.js:4081
Watcher.get @ vue.runtime.esm.js:4495
Watcher @ vue.runtime.esm.js:4484
el @ vue.runtime.esm.js:4088
Vue.$mount @ vue.runtime.esm.js:8459
init @ vue.runtime.esm.js:3137
(anonymous) @ vue.runtime.esm.js:6022
createElm @ vue.runtime.esm.js:5969
(anonymous) @ vue.runtime.esm.js:6521
Vue._update @ vue.runtime.esm.js:3960
updateComponent @ vue.runtime.esm.js:4081
Watcher.get @ vue.runtime.esm.js:4495
Watcher @ vue.runtime.esm.js:4484
el @ vue.runtime.esm.js:4088
Vue.$mount @ vue.runtime.esm.js:8459
init @ vue.runtime.esm.js:3137
(anonymous) @ vue.runtime.esm.js:6022
createElm @ vue.runtime.esm.js:5969
createChildren @ vue.runtime.esm.js:6097
createElm @ vue.runtime.esm.js:5998
(anonymous) @ vue.runtime.esm.js:6521
Vue._update @ vue.runtime.esm.js:3960
updateComponent @ vue.runtime.esm.js:4081
Watcher.get @ vue.runtime.esm.js:4495
Watcher @ vue.runtime.esm.js:4484
el @ vue.runtime.esm.js:4088
Vue.$mount @ vue.runtime.esm.js:8459
init @ vue.runtime.esm.js:3137
(anonymous) @ vue.runtime.esm.js:6022
createElm @ vue.runtime.esm.js:5969
createChildren @ vue.runtime.esm.js:6097
createElm @ vue.runtime.esm.js:5998
createChildren @ vue.runtime.esm.js:6097
createElm @ vue.runtime.esm.js:5998
createChildren @ vue.runtime.esm.js:6097
createElm @ vue.runtime.esm.js:5998
(anonymous) @ vue.runtime.esm.js:6521
Vue._update @ vue.runtime.esm.js:3960
updateComponent @ vue.runtime.esm.js:4081
Watcher.get @ vue.runtime.esm.js:4495
Watcher @ vue.runtime.esm.js:4484
el @ vue.runtime.esm.js:4088
Vue.$mount @ vue.runtime.esm.js:8459
init @ vue.runtime.esm.js:3137
(anonymous) @ vue.runtime.esm.js:6022
createElm @ vue.runtime.esm.js:5969
createChildren @ vue.runtime.esm.js:6097
createElm @ vue.runtime.esm.js:5998
createChildren @ vue.runtime.esm.js:6097
createElm @ vue.runtime.esm.js:5998
(anonymous) @ vue.runtime.esm.js:6521
Vue._update @ vue.runtime.esm.js:3960
updateComponent @ vue.runtime.esm.js:4081
Watcher.get @ vue.runtime.esm.js:4495
Watcher @ vue.runtime.esm.js:4484
el @ vue.runtime.esm.js:4088
Vue.$mount @ vue.runtime.esm.js:8459
init @ vue.runtime.esm.js:3137
(anonymous) @ vue.runtime.esm.js:6022
createElm @ vue.runtime.esm.js:5969
(anonymous) @ vue.runtime.esm.js:6521
Vue._update @ vue.runtime.esm.js:3960
updateComponent @ vue.runtime.esm.js:4081
Watcher.get @ vue.runtime.esm.js:4495
Watcher @ vue.runtime.esm.js:4484
el @ vue.runtime.esm.js:4088
Vue.$mount @ vue.runtime.esm.js:8459
init @ vue.runtime.esm.js:3137
(anonymous) @ vue.runtime.esm.js:6022
createElm @ vue.runtime.esm.js:5969
(anonymous) @ vue.runtime.esm.js:6521
Vue._update @ vue.runtime.esm.js:3960
updateComponent @ vue.runtime.esm.js:4081
Watcher.get @ vue.runtime.esm.js:4495
Watcher @ vue.runtime.esm.js:4484
el @ vue.runtime.esm.js:4088
Vue.$mount @ vue.runtime.esm.js:8459
init @ vue.runtime.esm.js:3137
merged @ vue.runtime.esm.js:3322
(anonymous) @ vue.runtime.esm.js:6022
createElm @ vue.runtime.esm.js:5969
(anonymous) @ vue.runtime.esm.js:6560
Vue._update @ vue.runtime.esm.js:3963
updateComponent @ vue.runtime.esm.js:4081
Watcher.get @ vue.runtime.esm.js:4495
Watcher.run @ vue.runtime.esm.js:4570
flushSchedulerQueue @ vue.runtime.esm.js:4326
(anonymous) @ vue.runtime.esm.js:1989
flushCallbacks @ vue.runtime.esm.js:1915
Promise.then (async)
timerFunc @ vue.runtime.esm.js:1942
nextTick @ vue.runtime.esm.js:1999
(anonymous) @ actions.js:301
loading @ actions.js:299
(anonymous) @ vuex.esm.js:792
Store.dispatch @ vuex.esm.js:457
Store.dispatch @ vuex.esm.js:347
(anonymous) @ handlers.js:195
handler @ planLessonsRoutes.js:163
value @ router.js:22
iterator @ vue-router.esm.js:2362
step @ vue-router.esm.js:2087
(anonymous) @ vue-router.esm.js:2088
(anonymous) @ vue-router.esm.js:2384
Promise.then (async)
(anonymous) @ app.js:44
iterator @ vue-router.esm.js:2362
step @ vue-router.esm.js:2087
step @ vue-router.esm.js:2091
runQueue @ vue-router.esm.js:2095
History.confirmTransition @ vue-router.esm.js:2392
History.transitionTo @ vue-router.esm.js:2260
HashHistory.push @ vue-router.esm.js:2715
(anonymous) @ vue-router.esm.js:3036
VueRouter.push @ vue-router.esm.js:3035
onPreviewClick @ ReportsLessonResourceLearnerListPage.vue:207
invokeWithErrorHandling @ vue.runtime.esm.js:1863
invoker @ vue.runtime.esm.js:2188
invokeWithErrorHandling @ vue.runtime.esm.js:1863
Vue.$emit @ vue.runtime.esm.js:3903
click @ ReportsResourceHeader.vue?1595:48
invokeWithErrorHandling @ vue.runtime.esm.js:1863
invoker @ vue.runtime.esm.js:2188
invokeWithErrorHandling @ vue.runtime.esm.js:1863
Vue.$emit @ vue.runtime.esm.js:3903
handleClick @ KButton.vue:179
invokeWithErrorHandling @ vue.runtime.esm.js:1863
invoker @ vue.runtime.esm.js:2188
original._wrapper @ vue.runtime.esm.js:6961
vue.runtime.esm.js:1897 DOMException: Failed to execute 'appendChild' on 'Node': This node type does not support this method.
    at attachResizeEvent (http://127.0.0.1:8080/static/kolibri.core.default_frontend/kolibri.core.default_frontend-0.16.0.dev0%2Bgit.20221217152023.js:2:408163)
    at http://127.0.0.1:8080/static/kolibri.core.default_frontend/kolibri.core.default_frontend-0.16.0.dev0%2Bgit.20221217152023.js:2:409852
    at forEachElement (http://127.0.0.1:8080/static/kolibri.core.default_frontend/kolibri.core.default_frontend-0.16.0.dev0%2Bgit.20221217152023.js:2:406225)
    at new ResizeSensor (http://127.0.0.1:8080/static/kolibri.core.default_frontend/kolibri.core.default_frontend-0.16.0.dev0%2Bgit.20221217152023.js:2:409813)
    at Sub.mounted (http://127.0.0.1:8080/static/kolibri.core.default_frontend/kolibri.core.default_frontend-0.16.0.dev0%2Bgit.20221217152023.js:2:1681004)
    at invokeWithErrorHandling (http://127.0.0.1:8080/static/kolibri.core.default_frontend/kolibri.core.default_frontend-0.16.0.dev0%2Bgit.20221217152023.js:2:659972)
    at callHook (http://127.0.0.1:8080/static/kolibri.core.default_frontend/kolibri.core.default_frontend-0.16.0.dev0%2Bgit.20221217152023.js:2:684148)
    at Object.insert (http://127.0.0.1:8080/static/kolibri.core.default_frontend/kolibri.core.default_frontend-0.16.0.dev0%2Bgit.20221217152023.js:2:675474)
    at invokeInsertHook (http://127.0.0.1:8080/static/kolibri.core.default_frontend/kolibri.core.default_frontend-0.16.0.dev0%2Bgit.20221217152023.js:2:734815)
    at Vue.__patch__ (http://127.0.0.1:8080/static/kolibri.core.default_frontend/kolibri.core.default_frontend-0.16.0.dev0%2Bgit.20221217152023.js:2:737435)
logError @ vue.runtime.esm.js:1897
globalHandleError @ vue.runtime.esm.js:1888
handleError @ vue.runtime.esm.js:1848
invokeWithErrorHandling @ vue.runtime.esm.js:1871
callHook @ vue.runtime.esm.js:4235
insert @ vue.runtime.esm.js:3158
invokeInsertHook @ vue.runtime.esm.js:6390
(anonymous) @ vue.runtime.esm.js:6609
Vue._update @ vue.runtime.esm.js:3963
updateComponent @ vue.runtime.esm.js:4081
Watcher.get @ vue.runtime.esm.js:4495
Watcher.run @ vue.runtime.esm.js:4570
flushSchedulerQueue @ vue.runtime.esm.js:4326
(anonymous) @ vue.runtime.esm.js:1989
flushCallbacks @ vue.runtime.esm.js:1915
Promise.then (async)
timerFunc @ vue.runtime.esm.js:1942
nextTick @ vue.runtime.esm.js:1999
(anonymous) @ actions.js:301
loading @ actions.js:299
(anonymous) @ vuex.esm.js:792
Store.dispatch @ vuex.esm.js:457
Store.dispatch @ vuex.esm.js:347
(anonymous) @ handlers.js:195
handler @ planLessonsRoutes.js:163
value @ router.js:22
iterator @ vue-router.esm.js:2362
step @ vue-router.esm.js:2087
(anonymous) @ vue-router.esm.js:2088
(anonymous) @ vue-router.esm.js:2384
Promise.then (async)
(anonymous) @ app.js:44
iterator @ vue-router.esm.js:2362
step @ vue-router.esm.js:2087
step @ vue-router.esm.js:2091
runQueue @ vue-router.esm.js:2095
History.confirmTransition @ vue-router.esm.js:2392
History.transitionTo @ vue-router.esm.js:2260
HashHistory.push @ vue-router.esm.js:2715
(anonymous) @ vue-router.esm.js:3036
VueRouter.push @ vue-router.esm.js:3035
onPreviewClick @ ReportsLessonResourceLearnerListPage.vue:207
invokeWithErrorHandling @ vue.runtime.esm.js:1863
invoker @ vue.runtime.esm.js:2188
invokeWithErrorHandling @ vue.runtime.esm.js:1863
Vue.$emit @ vue.runtime.esm.js:3903
click @ ReportsResourceHeader.vue?1595:48
invokeWithErrorHandling @ vue.runtime.esm.js:1863
invoker @ vue.runtime.esm.js:2188
invokeWithErrorHandling @ vue.runtime.esm.js:1863
Vue.$emit @ vue.runtime.esm.js:3903
handleClick @ KButton.vue:179
invokeWithErrorHandling @ vue.runtime.esm.js:1863
invoker @ vue.runtime.esm.js:2188
original._wrapper @ vue.runtime.esm.js:6961
xhr.js:220          GET http://127.0.0.1:8080/api/content/contentnode//?contentCacheKey=1671627010 404 (Not Found)
(anonymous) @ xhr.js:220
module.exports @ xhr.js:16
module.exports @ dispatchRequest.js:56
Promise.then (async)
Axios.request @ Axios.js:89
(anonymous) @ bind.js:9
client @ client.js:105
value @ api-resource.js:1047
fetchModel @ contentNode.js:134
_prepLessonContentPreview @ handlers.js:246
(anonymous) @ handlers.js:196
Promise.then (async)
(anonymous) @ handlers.js:195
handler @ planLessonsRoutes.js:163
value @ router.js:22
iterator @ vue-router.esm.js:2362
step @ vue-router.esm.js:2087
(anonymous) @ vue-router.esm.js:2088
(anonymous) @ vue-router.esm.js:2384
Promise.then (async)
(anonymous) @ app.js:44
iterator @ vue-router.esm.js:2362
step @ vue-router.esm.js:2087
step @ vue-router.esm.js:2091
runQueue @ vue-router.esm.js:2095
History.confirmTransition @ vue-router.esm.js:2392
History.transitionTo @ vue-router.esm.js:2260
HashHistory.push @ vue-router.esm.js:2715
(anonymous) @ vue-router.esm.js:3036
VueRouter.push @ vue-router.esm.js:3035
onPreviewClick @ ReportsLessonResourceLearnerListPage.vue:207
invokeWithErrorHandling @ vue.runtime.esm.js:1863
invoker @ vue.runtime.esm.js:2188
invokeWithErrorHandling @ vue.runtime.esm.js:1863
Vue.$emit @ vue.runtime.esm.js:3903
click @ ReportsResourceHeader.vue?1595:48
invokeWithErrorHandling @ vue.runtime.esm.js:1863
invoker @ vue.runtime.esm.js:2188
invokeWithErrorHandling @ vue.runtime.esm.js:1863
Vue.$emit @ vue.runtime.esm.js:3903
handleClick @ KButton.vue:179
invokeWithErrorHandling @ vue.runtime.esm.js:1863
invoker @ vue.runtime.esm.js:2188
original._wrapper @ vue.runtime.esm.js:6961
vue.runtime.esm.js:1897 TypeError: Cannot read properties of undefined (reading 'detach')
    at Sub.beforeDestroy (KResponsiveElementMixin.js:35:33)
    at invokeWithErrorHandling (vue.runtime.esm.js:1863:57)
    at callHook (vue.runtime.esm.js:4235:7)
    at Vue.$destroy (vue.runtime.esm.js:3993:5)
    at destroy (vue.runtime.esm.js:3178:27)
    at invokeDestroyHook (vue.runtime.esm.js:6158:59)
    at Sub.__patch__ (vue.runtime.esm.js:6511:30)
    at Vue.$destroy (vue.runtime.esm.js:4016:8)
    at destroy (vue.runtime.esm.js:3178:27)
    at invokeDestroyHook (vue.runtime.esm.js:6158:59)
logError @ vue.runtime.esm.js:1897
globalHandleError @ vue.runtime.esm.js:1888
handleError @ vue.runtime.esm.js:1848
invokeWithErrorHandling @ vue.runtime.esm.js:1871
callHook @ vue.runtime.esm.js:4235
Vue.$destroy @ vue.runtime.esm.js:3993
destroy @ vue.runtime.esm.js:3178
invokeDestroyHook @ vue.runtime.esm.js:6158
(anonymous) @ vue.runtime.esm.js:6511
Vue.$destroy @ vue.runtime.esm.js:4016
destroy @ vue.runtime.esm.js:3178
invokeDestroyHook @ vue.runtime.esm.js:6158
invokeDestroyHook @ vue.runtime.esm.js:6163
(anonymous) @ vue.runtime.esm.js:6511
Vue.$destroy @ vue.runtime.esm.js:4016
destroy @ vue.runtime.esm.js:3178
invokeDestroyHook @ vue.runtime.esm.js:6158
invokeDestroyHook @ vue.runtime.esm.js:6163
invokeDestroyHook @ vue.runtime.esm.js:6163
invokeDestroyHook @ vue.runtime.esm.js:6163
(anonymous) @ vue.runtime.esm.js:6511
Vue.$destroy @ vue.runtime.esm.js:4016
destroy @ vue.runtime.esm.js:3178
invokeDestroyHook @ vue.runtime.esm.js:6158
invokeDestroyHook @ vue.runtime.esm.js:6163
removeVnodes @ vue.runtime.esm.js:6174
(anonymous) @ vue.runtime.esm.js:6279
patchVnode @ vue.runtime.esm.js:6363
(anonymous) @ vue.runtime.esm.js:6526
Vue._update @ vue.runtime.esm.js:3963
updateComponent @ vue.runtime.esm.js:4081
Watcher.get @ vue.runtime.esm.js:4495
Watcher.run @ vue.runtime.esm.js:4570
flushSchedulerQueue @ vue.runtime.esm.js:4326
(anonymous) @ vue.runtime.esm.js:1989
flushCallbacks @ vue.runtime.esm.js:1915
Promise.then (async)
timerFunc @ vue.runtime.esm.js:1942
nextTick @ vue.runtime.esm.js:1999
(anonymous) @ vue.runtime.esm.js:4418
Watcher.update @ vue.runtime.esm.js:4560
Dep.notify @ vue.runtime.esm.js:730
set @ vue.runtime.esm.js:1055
CORE_SET_ERROR @ mutations.js:16
(anonymous) @ vuex.esm.js:785
(anonymous) @ vuex.esm.js:407
(anonymous) @ vuex.esm.js:406
Store._withCommit @ vuex.esm.js:565
Store.commit @ vuex.esm.js:405
Store.commit @ vuex.esm.js:350
handleError @ actions.js:66
handleApiError @ actions.js:90
(anonymous) @ vuex.esm.js:792
Store.dispatch @ vuex.esm.js:457
Store.dispatch @ vuex.esm.js:347
(anonymous) @ handlers.js:264
Promise.then (async)
_prepLessonContentPreview @ handlers.js:246
(anonymous) @ handlers.js:196
Promise.then (async)
(anonymous) @ handlers.js:195
handler @ planLessonsRoutes.js:163
value @ router.js:22
iterator @ vue-router.esm.js:2362
step @ vue-router.esm.js:2087
(anonymous) @ vue-router.esm.js:2088
(anonymous) @ vue-router.esm.js:2384
Promise.then (async)
(anonymous) @ app.js:44
iterator @ vue-router.esm.js:2362
step @ vue-router.esm.js:2087
step @ vue-router.esm.js:2091
runQueue @ vue-router.esm.js:2095
History.confirmTransition @ vue-router.esm.js:2392
History.transitionTo @ vue-router.esm.js:2260
HashHistory.push @ vue-router.esm.js:2715
(anonymous) @ vue-router.esm.js:3036
VueRouter.push @ vue-router.esm.js:3035
onPreviewClick @ ReportsLessonResourceLearnerListPage.vue:207
invokeWithErrorHandling @ vue.runtime.esm.js:1863
invoker @ vue.runtime.esm.js:2188
invokeWithErrorHandling @ vue.runtime.esm.js:1863
Vue.$emit @ vue.runtime.esm.js:3903
click @ ReportsResourceHeader.vue?1595:48
invokeWithErrorHandling @ vue.runtime.esm.js:1863
invoker @ vue.runtime.esm.js:2188
invokeWithErrorHandling @ vue.runtime.esm.js:1863
Vue.$emit @ vue.runtime.esm.js:3903
handleClick @ KButton.vue:179
invokeWithErrorHandling @ vue.runtime.esm.js:1863
invoker @ vue.runtime.esm.js:2188
original._wrapper @ vue.runtime.esm.js:6961
:8080/en/coach/#/?la…4a83d19ac067dff60:1 Uncaught (in promise) AxiosError {message: 'Request failed with status code 404', name: 'AxiosError', code: 'ERR_BAD_REQUEST', config: {…}, request: XMLHttpRequest, …}
XMLHttpRequest.send (async)
(anonymous) @ xhr.js:220
module.exports @ xhr.js:16
module.exports @ dispatchRequest.js:56
Promise.then (async)
Axios.request @ Axios.js:89
(anonymous) @ bind.js:9
client @ client.js:105
value @ api-resource.js:1047
fetchModel @ contentNode.js:134
_prepLessonContentPreview @ handlers.js:246
(anonymous) @ handlers.js:196
Promise.then (async)
(anonymous) @ handlers.js:195
handler @ planLessonsRoutes.js:163
value @ router.js:22
iterator @ vue-router.esm.js:2362
step @ vue-router.esm.js:2087
(anonymous) @ vue-router.esm.js:2088
(anonymous) @ vue-router.esm.js:2384
Promise.then (async)
(anonymous) @ app.js:44
iterator @ vue-router.esm.js:2362
step @ vue-router.esm.js:2087
step @ vue-router.esm.js:2091
runQueue @ vue-router.esm.js:2095
History.confirmTransition @ vue-router.esm.js:2392
History.transitionTo @ vue-router.esm.js:2260
HashHistory.push @ vue-router.esm.js:2715
(anonymous) @ vue-router.esm.js:3036
VueRouter.push @ vue-router.esm.js:3035
onPreviewClick @ ReportsLessonResourceLearnerListPage.vue:207
invokeWithErrorHandling @ vue.runtime.esm.js:1863
invoker @ vue.runtime.esm.js:2188
invokeWithErrorHandling @ vue.runtime.esm.js:1863
Vue.$emit @ vue.runtime.esm.js:3903
click @ ReportsResourceHeader.vue?1595:48
invokeWithErrorHandling @ vue.runtime.esm.js:1863
invoker @ vue.runtime.esm.js:2188
invokeWithErrorHandling @ vue.runtime.esm.js:1863
Vue.$emit @ vue.runtime.esm.js:3903
handleClick @ KButton.vue:179
invokeWithErrorHandling @ vue.runtime.esm.js:1863
invoker @ vue.runtime.esm.js:2188
original._wrapper @ vue.runtime.esm.js:6961
```

## Usage Details
Windows 10, Ubuntu - Chrome, Firefox

--- a/kolibri/plugins/coach/assets/src/views/reports/ReportsLessonResourceLearnerListPage.vue
+++ b/kolibri/plugins/coach/assets/src/views/reports/ReportsLessonResourceLearnerListPage.vue
@@ -208,7 +208,7 @@
           this.$router.getRoute(
             'RESOURCE_CONTENT_PREVIEW',
             {
-              contentId: this.resource.node_id,
+              contentId: this.resource.id,
             },
             {
               last: lastPage,

